great_expectations/validator/validator.py

```diff
@@ -1140,13 +1144,16 @@ def _generate_metric_dependency_subgraphs_for_each_expectation_configuration(
                 else:
                     raise err
 
-        return evrs, processed_configurations
+        return expectation_validation_graphs, evrs, processed_configurations
 
     def _generate_suite_level_graph_from_expectation_level_sub_graphs(
         self,
         expectation_validation_graphs: List[ExpectationValidationGraph],
     ) -> ValidationGraph:
         # Collect edges from all expectation-level sub-graphs and incorporate them under common suite-level graph.
+        if expectation_validation_graphs is None:
```

Comment:
According to the method signature this cannot be `None`.

Reply:
Incorporated -- thanks!